templates/quickstart-jira-dc.template.yaml

```diff
@@ -451,6 +454,10 @@ Parameters:
     Default: "aws_jira_dc_node.yml"
     Type: String
     Description: The Ansible playbook to invoke to initialise the Jira node on first start.
+  DeploymentAutomationCustomParams:
+    Default: ""
+    Type: String
+    Description: Additional command-line options for the `ansible-playbook` command. See the repository README for more details. (Optional)
```

Comment:
Should we include a link to the repository or the readme here?

Reply:
The link is effectively the parameter DeploymentAutomationRepository.

Reply:
Oh right 🤦‍♂ However, do we need to call that out explicitly in case a user can't put those two together. cc @ddomingorh

Reply:
I agree I think the immediate reaction would be to look on README of this repository. Which might be also an option - to add it here but I think in general we are trying to keep the documentation in the deployment guide and automation repository. I think right now we don't have any documentation paragraph about Ansible in our guide. Maybe it would be good to add it there so people know about it and also are able to customize? I have mentioned it a webinar recording to spread a word about it.

Reply:
I'll raise a ticket to add it to the guide.